go.mod

```diff
@@ -4,31 +4,31 @@ go 1.19
 
 require (
 	github.com/Azure/azure-sdk-for-go v60.1.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.23
-	github.com/Azure/go-autorest/autorest/adal v0.9.17
+	github.com/Azure/go-autorest/autorest v0.11.27
+	github.com/Azure/go-autorest/autorest/adal v0.9.20
 	github.com/aws/aws-sdk-go v1.41.6
-	github.com/go-logr/logr v1.2.0
-	github.com/google/go-cmp v0.5.6
+	github.com/go-logr/logr v1.2.3
+	github.com/google/go-cmp v0.5.8
 	github.com/infobloxopen/infoblox-go-client v1.1.1
 	github.com/miekg/dns v1.0.14
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.18.1
-	github.com/openshift/api v0.0.0-20211019100638-b2cbe79f2e4b
+	github.com/onsi/gomega v1.20.1
+	github.com/openshift/api v0.0.0-20220906163444-2df055c101a3
```

Comment:
is this changed to specific commit ?

Reply:
This is the golang module versioning in case the tagging format is not used. It defaults to the date + the checksum of the commit. I just updated to the openshift api which used 0.25 K8S to be aligned with the K8S bump.